FRW/Services/Manager/UserManager.swift

```diff
@@ -310,27 +336,36 @@ extension UserManager {
             loginAnonymousIfNeeded()
             throw LLError.restoreLoginFailed
         }
+        let secureKey = try SecureEnclaveKey.wallet(id: userId)
         
-        guard let publicData = try WallectSecureEnclave.Store.fetch(by: userId), !publicData.isEmpty else {
-            throw LLError.restoreLoginFailed
-        }
-        
-        let sec = try WallectSecureEnclave(privateKey: publicData)
-    
-        guard let signData = token.AddUserMessage(),
-              let publicKey = sec.key.publickeyValue,
+        guard let signData = token.addUserMessage(),
+              let publicKey = try secureKey.publicKey()?.hexValue,
               !publicKey.isEmpty
         else {
             throw LLError.signFailed
         }
-        let signature = try sec.sign(data: signData).hexValue
+        let signature = try secureKey.sign(data: signData, hashAlgo: .SHA2_256)
+
+//        guard let publicData = try WallectSecureEnclave.Store.fetch(by: userId), !publicData.isEmpty else {
+//            throw LLError.restoreLoginFailed
+//        }
+        
+//        let sec = try WallectSecureEnclave(privateKey: publicData)
+    
+//        guard let signData = token.AddUserMessage(),
+//              let publicKey = sec.key.publickeyValue,
+//              !publicKey.isEmpty
+//        else {
+//            throw LLError.signFailed
+//        }
+//        let signature = try sec.sign(data: signData).hexValue
```

Comment:
If this code is commented but left for a specific reason, it would be better to describe why, otherwise I guess it can be deleted